### ISSUE TITLE
Naming of categories and pages.

### ISSUE THREAD

**Issue by martinmoene** [accu-org/essential-tools#5]
Seeing the names of the categories on the wiki Home page and the page (file) names, I'm wondering if there's is useful naming scheme for them.

For the entries in the table of contents I think the word _tools_ is redundant, it's quite clear we're talking about tools and it would be a bit too much too see the word appear almost every line.

However, I think it's different for page (file) names: there it's  appropriate to be reminded we're talking about tools and to use e.g. _Profiling and Code Coverage Tools_.

So I propose the following:
- for table of contents and sidebar omit the word _tools_, e.g. use:  
  _Profiling and Code Coverage_.
- for page (file) names do use the word _tools_, e.g. use:  
  _Profiling and Code Coverage Tools_.

A consequence would be -I think- that categories would be named like (think: Tools for -)
- Backup
- Compilation
- Editing
- Profiling and Code Coverage
- Static Analysis
- Testing

instead of :
- Backup Tools
- Compilers
- Editors and IDEs
- Profiling and Code Coverage Tools
- Static Analysers / Static Analysis Tools
- Testing

What about the use of case? The [Essential books](https://github.com/accu-org/essential-books/wiki) page uses [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case).

What do _you_ think?

**Comment by rollbear** [comment]
Generally very good, although I'm a bit uncertain about "Editing," given that an IDE is so much more than just an editor.

**Comment by codalogic** [comment]
I agree with the desire not to keep repeating "Tools", but I'm not sure that the proposed solution is the way to deal with it.  I'd be happy with topic headings like:
- Backup
- Compilers
- Editors and IDEs
- Profiling and Code Coverage
- Static Analysers
- Testing

I think the rule is: "Don't use the word 'Tools'"!

**Comment by martinmoene** [comment]
Although these phrases are not similar in how they describe the category[1] &ndash;something to strive for in a list, it _is_ perhaps the pragmatic way to go ;-)

[1]  some describe a _thing_, some describe an _activity_.

**Comment by codalogic** [comment]
:+1: I think it's the least jarring option.

Perhaps the rule is:

if( is_activity )
   append("Tools"); // To make it a thing
remove("Tools"); // To avoid tedious repetition

:-)
